mergebot_poller.py

```diff
@@ -75,8 +76,7 @@ def poll_github(self):
         """Polls Github for PRs, verifies, then searches them for commands.
         """
         # Kick off an SCM merger
-        merger_args = (self.work_queue, self.known_work, self.github_helper,)
-        merger = Thread(target=merge.merge_git, args=merger_args)
+        merger = Thread(target=self.merger.merge)
```

Comment:
For clarity, can we call this mergerThread or something like that? I know merge vs. self.merger distinguishes them, but it would be good to be more explicit about which is a thread and which is a merger.